CDP4CrossViewEditor.Tests/Assemblers/CrossviewPMeanArrayAssemblerTestFixture.cs

```diff
@@ -0,0 +1,701 @@
+                    ShortName = "W"
+                },
+                Owner = this.domain,
+                StateDependence = actualList,
+                ValueSet =
+                {
+                    new ParameterValueSet(Guid.NewGuid(), this.assembler.Cache, this.credentials.Uri)
+                    {
+                        Manual = new ValueArray<string>(new List<string> { "-" }),
+                        ValueSwitch = ParameterSwitchKind.MANUAL,
+                        ActualOption = option1,
+                        ActualState = actualFiniteState
+                    },
+                    new ParameterValueSet(Guid.NewGuid(), this.assembler.Cache, this.credentials.Uri)
+                    {
+                        Manual = new ValueArray<string>(new List<string> { "-" }),
+                        ValueSwitch = ParameterSwitchKind.MANUAL,
+                        ActualOption = option2,
+                        ActualState = actualFiniteState
+                    }
+                }
+            };
+
+            this.parameterTypes.Add(P_redundancy);
+            this.parameterTypes.Add(P_on);
+            this.parameterTypes.Add(P_stby);
+            this.parameterTypes.Add(P_duty_cyc);
+            this.parameterTypes.Add(P_mean);
+
+            elementDefinition.Parameter.Add(this.pRedundancyActiveInternalParameter);
+            elementDefinition.Parameter.Add(this.pOnParameter);
+            elementDefinition.Parameter.Add(this.pStandByParameter);
+            elementDefinition.Parameter.Add(this.pDutyCycleParameter);
+            elementDefinition.Parameter.Add(this.pMeanParameter);
+
+            this.iteration.Element.Add(elementDefinition);
+
+            this.siteReferenceDataLibrary.ParameterType.Add(P_redundancy);
+            this.siteReferenceDataLibrary.ParameterType.Add(P_on);
+            this.siteReferenceDataLibrary.ParameterType.Add(P_stby);
+            this.siteReferenceDataLibrary.ParameterType.Add(P_duty_cyc);
+            this.siteReferenceDataLibrary.ParameterType.Add(P_mean);
+
+            this.session.Setup(x => x.OpenIterations).Returns(new Dictionary<Iteration, Tuple<DomainOfExpertise, Participant>>
+            {
+                {this.iteration, new Tuple<DomainOfExpertise, Participant>(this.domain, null)}
+            });
+
+            this.session.Setup(x => x.QuerySelectedDomainOfExpertise(It.IsAny<Iteration>())).Returns(this.domain);
+        }
+
+        [Test]
+        public void VerifyThatAssemblerCalculatesPMeanIfPDutyCycleAndPMeanAreProperlySet()
+        {
+            var crossviewSheetRowAssembler = new CrossviewSheetRowAssembler();
+            crossviewSheetRowAssembler.Assemble(this.elementDefinitions);
+            this.excelRows.AddRange(crossviewSheetRowAssembler.ExcelRows);
+
+            var arrayAssembler = new CrossviewArrayAssembler(this.excelRows, this.parameterTypes);
+
+            // The that power related parameters are present into the header
```

Comment:
Typo in comment "the that power related..."?

Given that this is repeated to many times, I would just remove this comment completely.